libs-scala/crypto/src/main/scala/com/daml/crypto/MessageDigestPrototype.scala

```diff
@@ -23,6 +24,9 @@ final class MessageDigestPrototype(val algorithm: String) {
       true
     } catch {
       case _: CloneNotSupportedException =>
+        logger.warn(
+          s"$algorithm.clone() is not supported. It might have implications on performance."
```

Comment:
Maybe you want to use `prototype.getClass.getName` here instead of `algorithm`.

Reply:
ah you're right, sure